.golangci.yml

```diff
@@ -1,23 +0,0 @@
-run:
```

Comment:
Not sure how golint work but if it covers the tests and what is now under `profile-build` then I'd keep this one and the related Earthfile lines

Reply:
do we really care about linting the test files? just asking because to me it makes not much sense to lint test files. And the profile-build files is...just one file which runs something from the sdk, kind of a frontend to it, which zero real code.

Linting should be done in the kairos-agent repo instead (its already in there)
Test files should need no linting as they are not for any external use or anything, just run the tests which is the important part.

I can restore it if you want?

Reply:
nah, all good, if we introduce some other piece of code we can bring it back